Makefile

```diff
@@ -31,7 +31,7 @@ KIND_KUBERNETES_VERSION ?= 1.29.2
 GATEKEEPER_VERSION ?= 3.16.0
 DAPR_VERSION ?= 1.12.5
 COSIGN_VERSION ?= 2.2.3
-NOTATION_VERSION ?= 1.1.0
+NOTATION_VERSION ?= 1.2.0-beta.1
```

Comment:
i m guessing we will leave this PR open , and wait for notation 1.2?

Reply:
This is targeting a separate branch preparing for the tsa bump only, so it is save to merge.